pkg/gfx/framebuffer.go

```diff
@@ -0,0 +1,47 @@
+package gfx
+
+import (
+	"github.com/go-gl/gl/v2.1/gl"
+	"github.com/gregjohnson2017/tabula-editor/pkg/log"
+)
+
+type FrameBuffer struct {
+	id  uint32
+	tex Texture
+}
+
+const ErrFrameBuffer log.ConstErr = "incomplete framebuffer"
+
+func NewFrameBuffer(width, height int32) (FrameBuffer, error) {
```

Comment:
This file is not being used.

Reply:
What do you mean? It's being used for exporting the canvas in view.go on line 363

Reply:
Ok, I see it now. GitHub was hiding the diff due to the size of it.